atd-vzd/README.md

```diff
@@ -35,7 +35,7 @@ We use a standard metabase container for business intelligence analytics as well
 
 2. Once you changes have been made, copy the SQL statement that modified the table to our [migrations folder](/migrations) for version control. Example: 
  ```sql
-  ALTER TABLE "public"."atd_txdot_crashes" ADD COLUMN "micromobility_device_flag" varchar(1) NOT NULL DEFAULT 'N';
```

Comment:
Wow - what are the odds that this exact column was used in this doc!

Reply:
i know!! we'll need to rework the documentation of course but this was bothering me heh